### ISSUE TITLE
Translation does not work v.1.1

### ISSUE THREAD

**Issue by Clarensio** [DavidXanatos/wumgr#103]
The translation (at least relating to Italian) is not displayed / does not work

**Comment by drellmark** [comment]
Yes, that's right for me, too (Windows 10 installed with English media and added German language later).

Looking in the code I got the bug.
In Translate.cs the user interface language is retrieved by a wrong query.

Current (line 19):
CultureInfo ci = CultureInfo.InstalledUICulture;
Should be:
CultureInfo ci = CultureInfo.CurrentUICulture;

You can fix it for yourself by adding the line "Lang=it" into your wumgr.ini in the Options-section
